### ISSUE TITLE
Document supported input types for StackingCVClassifier

### ISSUE THREAD

**Issue by hengji-liu** [rasbt/mlxtend#197]
in `stacking_cv_classification.py`, on line 161, `model.fit(X[train_index], y[train_index])`
`train_index` is an ndarray instead of an int,
which causes `TypeError: only integer scalar arrays can be converted to a scalar index`
This happens when I
```
sclf = StackingCVClassifier(classifiers=[ada, rfc, etc], meta_classifier=lr)
sclf.fit(X, y)
```
where X is a list of lists and y is a list.
And it will be Pandas KeyError if I pass X and y as Dataframes

**Comment by rasbt** [comment]
Thanks for the note! Yes, that's correct, `train_idx` is currently a numpy array. Unfortunately, we can't use an `int` here because it's selecting multiple records at once. I.e., it's selecting all the training rows from the input arrays `X` and `y`. 

Like you said, the issue is basically the following: 

```
>>> a = np.array([1, 2, 3, 4, 5, 6])
>>> a[np.array([2, 5])]
array([3, 6])

>>> b = pd.DataFrame(a)
>>> b[np.array([2, 5])]
... raise KeyError('%s not in index' % objarr[mask])

>>> c[np.array([2, 5])]
... TypeError: only integer scalar arrays can be converted to a scalar index
```

Not sure, but I think the best way to handle that would be to be more specific in the docstring documentation and raise more meaningful error messages?

Alternatively, the StackingCVClassifier could be modified to support lists of lists and pandas DataFrames, but this would involve additional checks for different data types. Generally, I think it's easier to simply pass a pandas DataFrame as `dataframe.values` or a list of lists as `np.array(list_of_list)`.

However, if you have good solution for supporting lists of lists and data frames in the fit function, I'd welcome a PR :).

**Comment by hengji-liu** [comment]
Thank you for your reply. `dataframe.values` works fine.
Given that the acceptable types are different from sklearn, can you please specify this in the document, so users can be aware.

**Comment by rasbt** [comment]
Sure, I will look into that. Thanks for brining it up!

**Comment by rasbt** [comment]
addressed in #198
